winui/Radial-Gauge/shape-pointer.md

```diff
@@ -113,6 +113,99 @@ this.Content = sfRadialGauge;
 
 ![WinUI Radial Gauge custom shape pointer](images/marker-pointers/winui-radial-gauge-custom-marker-pointer.png)
 
+## Shadow support
+
+The shadow can be applied by the `HasShadow` property.
```

Comment:
The shadow can be applied by using the `HasShadow` property.

Reply:
Changed the content as you mentioned.